src/theory/quantifiers/sygus/sygus_unif_rl.cpp

```diff
@@ -352,13 +366,11 @@ void SygusUnifRl::setConditions(Node e, const std::vector<Node>& conds)
 {
   std::map<Node, DecisionTreeInfo>::iterator it = d_stratpt_to_dt.find(e);
   Assert(it != d_stratpt_to_dt.end());
-  it->second.clearCondValues();
-  /* TODO
-  for (const Node& c : conds)
-  {
-    it->second.addCondValue(c);
-  }
-  */
+  // Clear previous trie
+  it->second.clearPointSeparator();
```

Comment:
Encapsulate this into it->second.setConditions(conds) and remove clearPointSeparator.